### PR TITLE
header -popup feature

### PR DESCRIPTION
**Pull request by ILAroy-611**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,11 +3,17 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@emotion/react": "^11.10.4",
+    "@emotion/styled": "^11.10.4",
```
Comment (bim240) [inline package.json:7]: These packages are not being used even once. What is the significance?

File: src/Components/Header/Popup/Popup.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    const handleChange = event => {
+        setUser({ ...user, [event.target.name]: event.target.value });
+    }
+    const handleLogin = event => {
+        event.preventDefault();
+        console.log(user);
+        sendUserInfo();
+    }
+
+    const sendUserInfo = async () => {
```
Comment (bim240) [inline src/Components/Header/Popup/Popup.js]: You are trying to call the function `sendUserInfo` before it is defined. Although react doesn't through any error but this is bad practice.

`sendUserInfo` doesn't give a message about what this function is doing. what you can do instead is change 

handleLogin -> handleSubmit
sendUserInfo -> handleLogin 

or anything that suits

**Review by bim240 — CHANGES_REQUESTED** [review]

**Comment by bim240** [comment]
css reset is missing. @ILAroy-611

File: package.json (hunk 1 above)
Comment (ILAroy-611) [reply package.json:7]: Hi Bim,

I have used Material UI icons and its pre-requisite to install above two packages in order to install-"@mui/icons-material": "^5.10.6" package.

File: src/Components/Header/Popup/Popup.js (hunk 2 above)
Comment (ILAroy-611) [reply src/Components/Header/Popup/Popup.js]: Corrected both things as suggested

**Pull request merged**